typing/typecore.ml

```diff
@@ -711,50 +711,87 @@ end) = struct
         warn lid.loc
           (Warnings.Disambiguated_name (get_name lbl))
 
+  let force : ('a, _) result -> 'a = function
```

Comment:
I would prefer a name that is not that close to `Lazy.force`. Maybe `force_error`?

Reply:
I used `force_error`, thanks.